Makefile

```diff
@@ -293,9 +297,13 @@ go-build-linux: $(BINARIES)
 run: docker-build kind-cluster kind-load kind-deploy #HELP Build the operator-controller then deploy it into a new kind cluster.
 
 .PHONY: docker-build
-docker-build: build-linux #EXHELP Build docker image for operator-controller with GOOS=linux and local GOARCH.
+docker-build: build-linux build-catalogd #EXHELP Build docker image for operator-controller with GOOS=linux and local GOARCH.
 	$(CONTAINER_RUNTIME) build -t $(IMG) -f Dockerfile ./bin/linux
 
+.PHONY: build-catalogd
+build-catalogd: ## Build the catalogd image
```

Comment:
Is there any reason to make this a separate task? I think I'd prefer to just directly put the command in the recipe of `docker-build`.

Reply:
And I suppose the same question applies to `generate-catalogd`. If we combine operator-controller's `manifests` and `generate` targets (or split catalogd's `generate` into two), can we do a direct `$(MAKE) -c catalog <target>` command in operator-controller's equivalent recipes?

Reply:
For the generate-catalogd we need because we need to ensure that we call it in all targets that we call generate/manifests for operator-controller.

However, for the build I agree we call it here.

Reply:
> For the generate-catalogd we need because we need to ensure that we call it in all targets that we call generate/manifests for operator-controller.

Yeah, I'm wondering if we can update operator-controller's Makefile `generate` and `manifests` to do the `$(MAKE) -c catalogd generate` essentially.

Then everywhere else in operator-controller's Makefile that already requires operator-controller's `generate` and/or `manifests` gets the catalog generation coming along for the ride.